src/app.css

```diff
@@ -4,6 +4,84 @@ code {
     Menlo,
     Monaco,
     Consolas,
-    'Courier New',
+    "Courier New",
     monospace;
 }
+
+/* Style The Dropdown Button */
```

Comment:
_note:  This PR does not contain all the content/files.  Therefore, I'll put my comments for the frontend here._

- [ ] Your app is very beautiful. On your readme, kindly add a screenshot to show it off and make it look more professional.

- [ ] I can't create reservations.  When I press the submit button, this happens:

### Firefox

![image](https://user-images.githubusercontent.com/57421823/208128872-274d4a86-3e61-48af-aa53-4e0720313ba5.png)

### Chrome

![image](https://user-images.githubusercontent.com/57421823/208129083-7b5afda9-67c4-4560-b7c1-2890946234a9.png)

### Console

![image](https://user-images.githubusercontent.com/57421823/208129140-c7f6b8cf-ff1b-4b6b-83e0-f7a13e1b295c.png)

Suggestion/early investigation:  It seems the frontend does not know about `current_user.id` and sends the POST the request just like that, and finally the server receives it as a string "current_user.id":

![image](https://user-images.githubusercontent.com/57421823/208129677-a1b2d3d5-9b49-4e71-b3a9-2fea58ae7663.png)


Kindly fix this bug, so your app works as intended. 💪